concepticondata/conceptlists/Anonby-2018-1500.tsv

```diff
@@ -0,0 +1,1501 @@
+ID	NUMBER	CONCEPTICON_ID	CONCEPTICON_GLOSS	ENGLISH	PERSIAN_ORTHOGRAPHY	PERSIAN_PHONOLOGICAL	MAIN_CATEGORY	SUB_CATEGORY1	SUB_CATEGORY2
+Anonby-2018-1500-1	1	1480	BODY	body	بدن	badan	Human physical being	Body parts	
+Anonby-2018-1500-2	2	2613	SKIN (HUMAN)	skin (of person)	پوست	pust	Human physical being	Body parts	
+Anonby-2018-1500-3	3	1256	HEAD	head	سر	sar	Human physical being	Body parts	Head
+Anonby-2018-1500-4	4	123	FOREHEAD	forehead	پیشانی	pišāni	Human physical being	Body parts	Head
+Anonby-2018-1500-5	5	1560	FACE	face	صورت	surat	Human physical being	Body parts	Head
+Anonby-2018-1500-6	6	1248	EYE	eye	چشم	češm / čašm	Human physical being	Body parts	Head
+Anonby-2018-1500-7	7	1173	EYEBROW	eyebrow	ابرو	abru	Human physical being	Body parts	Head
+Anonby-2018-1500-8	8	17	EYELID	eyelid	پلک	pelk	Human physical being	Body parts	Head
+Anonby-2018-1500-9	9	1540	EYELASH	eyelash	مژه	može	Human physical being	Body parts	Head
+Anonby-2018-1500-10	10	1658	PUPIL	pupil (of eye)	مردمک	mardomak	Human physical being	Body parts	Head
+Anonby-2018-1500-11	11	1221	NOSE	nose	دماغ، بینی	bini, damāġ 	Human physical being	Body parts	Head
+Anonby-2018-1500-12	12			bridge (of nose)	تیقه‌ی بینی، باریکه‌ی بینی	tiġe-ye bini, bārike-ye bini	Human physical being	Body parts	Head
+Anonby-2018-1500-13	13	1247	EAR	ear	گوش	guš	Human physical being	Body parts	Head
+Anonby-2018-1500-14	14	1730	CHEEK	cheek	گونه	gune	Human physical being	Body parts	Head
+Anonby-2018-1500-15	15	674	MOUTH	mouth	دهان	dahān	Human physical being	Body parts	Head
+Anonby-2018-1500-16	16	478	LIP	lip	لب	lab	Human physical being	Body parts	Head
+Anonby-2018-1500-17	17	1205	TONGUE	tongue	زبان	zabān	Human physical being	Body parts	Head
+Anonby-2018-1500-18	18	1380	TOOTH	tooth	دندان	dandān	Human physical being	Body parts	Head
+Anonby-2018-1500-19	19	1049	MOLAR TOOTH	molar tooth	دندانِ آسیاب	dandān-e āsiyāb	Human physical being	Body parts	Head
+Anonby-2018-1500-20	20	540	PALATE	palate	سقفِ دهان، کام	saġf-e dahān, kām	Human physical being	Body parts	Head
+Anonby-2018-1500-21	21	798	JAW	jaw	فک	fak	Human physical being	Body parts	Head
+Anonby-2018-1500-22	22	1510	CHIN	chin	چانه	čāne	Human physical being	Body parts	Head
+Anonby-2018-1500-23	23	1333	NECK	neck	گردن	gardan	Human physical being	Body parts	Head
+Anonby-2018-1500-24	24	1347	NAPE (OF NECK)	nape of neck	پسِ گردن، پشتِ گردن	pas-e gardan, pošt-e gardan	Human physical being	Body parts	Head
+Anonby-2018-1500-25	25	1346	THROAT	throat	گلو	galu / gelu	Human physical being	Body parts	Head
+Anonby-2018-1500-26	26	802	ADAM'S APPLE	voice box, larynx, Adam’s apple	حنجره	hanjere / hanjare	Human physical being	Body parts	Head
```

Comment:
Hm. I went for Adam's apple because it is the more visible, basic body part compared to the actual organ, but I see your point. Will unmap it, since Concepticon makes the distinction.